spec/query_builder_spec.rb

```diff
@@ -1,10 +1,8 @@
-# encoding: UTF-8
-
 require 'spec_helper'
 
 class TestQueryBuilder < ArelHelpers::QueryBuilder
   attr_accessor :params
-  alias_method :params?, :params
+  alias params? params
```

Comment:
Why change this to `alias`? I was under the impression `alias_method` was more flexible and generally preferred in the Ruby community these days.

Reply:
I don't recall making this change manually. Must have been my RuboCop auto-correct.

On that note, what do you think about introducing RuboCop to this project for a consistent style guide? I can do that in a separate PR.

Reply:
100% in favor of that :)